### PR TITLE
Add CommonJS support

### PR DESCRIPTION
**Pull request by sdebacker**
Here is the first step for a module in npm. Then you will have to do ```npm publish``` as mentioned in https://quickleft.com/blog/creating-and-publishing-a-node-js-module.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "name": "angular-ui-tree",
   "version": "2.7.0",
   "description": "An AngularJS UI component that can sort nested lists, provides drag & drop support and doesn't depend on jQuery",
+  "main": "index.js",
```
Comment (Voles) [inline package.json]: Is it possible to use `./dist/angular-ui-tree` directly as a value for the main file? Then we don't have to create an extra `index.js` file.

**Comment by sdebacker** [comment]
Yes, the just add ```module.exports = 'ui.tree';``` on to of it.

**Comment by Voles** [comment]
I would prefer that, WDYT? If you update your branch I can test the changes and merge!

**Comment by sdebacker** [comment]
I have too few experience with commonjs and browserify. BTW, this PR is not required to make ui-tree available to npm. If you just ‘npm publish’, it should be ok because the current package.json is valid. I think we can close this PR.

**Comment by Voles** [comment]
Thanks for your help, @sdebacker. I've published v2.7.0 as an NPM package.

**Comment by sdebacker** [comment]
Thanks !

**Pull request closed without merging**